### PR TITLE
Updated tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

### PR DESCRIPTION
**Pull request by shreekarSS**
- Fixed script issue for 1AZ aws ipi setup

Signed-off-by: Shreekar <sshreeka@redhat.com>

### CODE REVIEW

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 1]
```diff
@@ -18,34 +18,44 @@ class TestAddNode(ManageTest):
...
             machines = machine_utils.get_machinesets()
             for machine in machines:
-                before_replica_counts.update({machine: machine_utils.get_replica_count(machine)})
-            worker_nodes_before = helpers.get_worker_nodes()
-            logger.info(f'The worker nodes number before adding a new node is {len(worker_nodes_before)}')
+                before_replica_counts.update(
+                    {machine: machine_utils.get_replica_count(machine)}
+                )
+                logger.info(machine_utils.get_replica_count(machine))
+            logger.info(f'The worker nodes number before adding a new node is {len(helpers.get_worker_nodes())}')
```
Comment (ebenahar) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py]: Please break lines

**Comment by shreekarSS** [comment]
https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3840/testReport/ -- 1az

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 2]
```diff
@@ -18,34 +18,43 @@ class TestAddNode(ManageTest):
...
+                )
+                logger.info(machine_utils.get_replica_count(machine))
+            logger.info(f'The worker nodes number before {len(helpers.get_worker_nodes())}')
             after_replica_counts = dict()
+            total_count = 0
             for machine in machines:
-                machine_utils.add_node(machine, count=count)
-                after_replica_counts.update(({machine: machine_utils.get_replica_count(machine)}))
+                machine_utils.add_node(
+                    machine, count=machine_utils.get_replica_count(machine) + 1
```
Comment (ebenahar) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py]: should replica count be changed by adding a node?

Comment (shreekarSS) [reply tests/manage/z_cluster/cluster_expansion/test_node_expansion.py]: Add node happens when replica count is increased

**Comment by shreekarSS** [comment]
> https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3840/testReport/ -- 1az

https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/3949/ -- 3AZ

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 3]
```diff
@@ -18,34 +18,43 @@ class TestAddNode(ManageTest):
     """
     Automates adding worker nodes to the cluster while IOs
     """
+
     def test_add_node(self):
         """
         Test for adding worker nodes to the cluster while IOs
         """
         dt = config.ENV_DATA['deployment_type']
         if dt == 'ipi':
```
Comment (prsurve) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py]: you can use this marker @ipi_deployment_required

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py [hunk 4]
```diff
@@ -18,34 +18,43 @@ class TestAddNode(ManageTest):
...
-                machine_utils.add_node(machine, count=count)
-                after_replica_counts.update(({machine: machine_utils.get_replica_count(machine)}))
+                machine_utils.add_node(
+                    machine, count=machine_utils.get_replica_count(machine) + 1
+                )
+                after_replica_counts.update(
+                    ({machine: machine_utils.get_replica_count(machine)})
+                )
+                total_count += machine_utils.get_replica_count(machine)
+                logger.info(total_count)
```
Comment (prsurve) [inline tests/manage/z_cluster/cluster_expansion/test_node_expansion.py]: IF it's 3 az you can do add_node once in anyone of the az as per your code you are increasing from all az

File: tests/manage/z_cluster/cluster_expansion/test_node_expansion.py (hunk 3 above)
Comment (vasukulkarni) [reply tests/manage/z_cluster/cluster_expansion/test_node_expansion.py]: this is not part of his code change?

**Pull request merged**